client/src/components/Recycle/Recycle.js

```diff
@@ -11,7 +11,7 @@ const Recycle = () => {
     const fetchData = async () => {
         const response = await fetch(`https://api.ecohabit.org/recyclingcenters/country/${country}/postal/${postal}`);
         const data = await response.json();
-        setData(data);
+        setData(data.data);
```

Comment:
Shouldn't it be just `data` instead of `data.data`? second `data` exists in each element of the array, it looks like calling it data.data doesn't work before iterating (mapping) the response array.